plugins/destination/mssql/queries/tvp_test.go

```diff
@@ -69,7 +69,7 @@ func TestTVPAddType(t *testing.T) {
   [_cq_id] uniqueidentifier UNIQUE NOT NULL,
   [_cq_parent_id] uniqueidentifier,
   [_cq_source_name] nvarchar(4000),
-  [_cq_sync_time] datetimeoffset,
+  [_cq_sync_time] datetime2,
```

Comment:
`datetimeoffset` saves the offset so if we write a `time.Time` and then read it the time zone information gets added and we get a different value than the original (see comment in PR description and https://github.com/cloudquery/cloudquery/blob/ee4b2c01867b2140537ee4656d943d2e0cf2898b/plugins/destination/postgresql/client/types.go#L74

Reply:
Why don't we want the time zone?